src/Lecture1.hs

```diff
@@ -123,4 +145,14 @@ and lower than 6 elements (4, 5, 6, 7, 8 and 9).
 
 🕯 HINT: Use recursion to implement this function.
 -}
-lowerAndGreater n list = error "TODO"
+lowerAndGreater :: Int -> [Int] -> String
+lowerAndGreater n list = lowerAndGreater' n list 0 0
```

Comment:
I see a warning about ambiguous types here. This is because numeric constants like `0` are polymorphic and GHC doesn't know which type you want (`Int`, `Integer`, `Int8`, `Int16`, `Int32`, `Word`, `Natural` etc.) and defaults it to `Integer`.

This is not a problem in this particular case (besides the annoying warning) but usually, it's a good idea to specify types explicitly in such cases to avoid ambiguity 🙂